src/apps/recommender/Recommender.tsx

```diff
@@ -1,62 +1,32 @@
 import React, { FC, useEffect, useState } from "react";
-import { useGetLoansV2, useGetReservationsV2 } from "../../core/fbs/fbs";
 import {
   sortByLoanDate,
   sortByReservationDate
 } from "../../core/utils/helpers/general";
-import {
-  mapFBSLoanToLoanType,
-  mapFBSReservationToReservationType
-} from "../../core/utils/helpers/list-mapper";
 import { LoanType } from "../../core/utils/types/loan-type";
 import { ReservationType } from "../../core/utils/types/reservation-type";
 import InspirationRecommender from "./InspirationRecommender";
 import RecommendList from "./RecommendList";
+import useReservations from "../../core/utils/useReservations";
+import useLoans from "../../core/utils/useLoans";
 
 const Recommender: FC = () => {
   const [loanForRecommender, setLoanForRecommender] = useState<LoanType | null>(
     null
   );
   const [reservationForRecommender, setReservationForRecommender] =
     useState<ReservationType | null>(null);
-  const [physicalLoans, setPhysicalLoans] = useState<LoanType[] | null>(null);
-  const [physicalReservations, setPhysicalReservations] = useState<
-    ReservationType[] | null
-  >(null);
 
   const {
-    isSuccess: isSuccessFbsLoans,
-    error: fbsErrorLoans,
-    data: fbsLoans,
-    isLoading: fbsLoansLoading
-  } = useGetLoansV2();
+    fbs: { loans: physicalLoans, isLoading: fbsLoansLoading }
+  } = useLoans();
 
   const {
-    isSuccess: isSuccessFbsReservations,
-    error: fbsErrorReservations,
-    data: fbsReservations,
-    isLoading: fbsReservationsLoading
-  } = useGetReservationsV2();
-
-  useEffect(() => {
-    if (fbsLoans) {
-      setPhysicalLoans(mapFBSLoanToLoanType(fbsLoans));
-    }
-    if (fbsErrorLoans && !isSuccessFbsLoans) {
-      setPhysicalLoans([]);
-    }
-  }, [fbsLoans, fbsErrorLoans, isSuccessFbsLoans]);
-
-  useEffect(() => {
-    if (fbsReservations) {
-      setPhysicalReservations(
-        mapFBSReservationToReservationType(fbsReservations)
-      );
-    }
-    if (fbsErrorReservations && !isSuccessFbsReservations) {
-      setPhysicalReservations([]);
+    fbs: {
+      reservations: physicalReservations,
+      isLoading: fbsReservationsLoading
     }
-  }, [fbsErrorReservations, fbsReservations, isSuccessFbsReservations]);
+  } = useReservations();
```

Comment:
This is basically a followup on #655 which uses the new hooks in the recommender component.